### PR TITLE
Inital tank layout

### PR DESCRIPTION
**Pull request by greatfireball**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cctb-it-wuerzburg/installation/pull/28" target="_blank">ReviewNinja

**Comment by greatfireball** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/cctb-it-wuerzburg/installation/pull-request/28/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=cctb-it-wuerzburg/installation)

File: final_installation/storage_server/102_create_user_homes.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+GROUPNAME=ecologicalmodeling
+for i in s216849 jus15cq
+do
+  ZFSDATASET=tank/"$GROUPNAME"/home/"$i"
+  sudo zfs create -o mountpoint=/new_home/"$i" "$ZFSDATASET"
+done
+
+GROUPNAME=evolutionarygenomics
+for i in ama42bf ark06eu jaf81qa s2046859
```
Comment (iimog) [inline final_installation/storage_server/102_create_user_homes.sh:24]: s2046859 looks like a typo to me. I only know s numbers with 6 digits, this one has 7. Is this correct? !fix

Comment (iimog) [inline final_installation/storage_server/102_create_user_homes.sh:24]: !fixed

Comment (iimog) [inline final_installation/storage_server/102_create_user_homes.sh:24]: War so im System, kann Typo sein aber ist halt so.

Comment (greatfireball) [reply final_installation/storage_server/102_create_user_homes.sh:24]: That was the number, the user gave me... Therefore, it need to be that number typo or not ;)

**Comment by greatfireball** [comment]
@iimog wenn du mir noch ein approved gibst, dann erlaubt mir pullapprove auch den Merge :)

**Pull request merged**